djl-serving/workflows/multi-model/README.md

```diff
@@ -0,0 +1,285 @@
+# DJLServing Multi-Model Workflow Example
+
+This is an example of how to serve multiple models in a DJLServing Workflow. 
+
+This example uses an image of soccer players, puts that through a PyTorch model to do object detection to get
+a player from the image, then define an "extractPerson" function to extract the person image, then feeds that
+resulting image to an MXNet model to do pose estimation.
+
+## Setup
+
+Before starting, you need to install DJLServing. You can find installation instructions
+[here](https://github.com/deepjavalibrary/djl-serving#installation).
+
+## Define the workflow
+
+First, we use the `function` folder to start building our own custom `WorkflowFunction`.
+The `WorkflowFunction` is a lambda function that can be run within a workflow.
+
+```java
```

Comment:
Instead of show the code here, we can just link to the code. We don't have to update both place if we make any changes

Reply:
Changed.